test/integration/CommunityEndpoints.test.js

```diff
@@ -0,0 +1,246 @@
+import assert from 'assert'
+
+import { Contract, providers, utils, Wallet } from 'ethers'
+
+import StreamrClient from '../../src'
+import * as Token from '../../contracts/TestToken.json'
+
+import config from './config'
+
+function sleep(ms) {
+    return new Promise((resolve) => {
+        setTimeout(resolve, ms)
+    })
+}
+
+describe('CommunityEndPoints', () => {
+    let community
+
+    let testProvider
+    let adminClient
+    let adminWallet
+
+    beforeAll(async () => {
+        testProvider = new providers.JsonRpcProvider(config.ethereumServerUrl)
+        const network = await testProvider.getNetwork().catch((e) => {
+            throw new Error(`Connecting to Ethereum failed, config = ${JSON.stringify(config)}`, e)
+        })
+        console.log('Connected to Ethereum network: ', JSON.stringify(network))
+
+        adminWallet = new Wallet(config.privateKey, testProvider)
+        adminClient = new StreamrClient({
+            auth: {
+                privateKey: adminWallet.privateKey
+            },
+            autoConnect: false,
+            autoDisconnect: false,
+            ...config.clientOptions,
+        })
+
+        console.log('beforeAll done')
+    })
+    beforeEach(async () => {
+        console.log('starting beforeEach')
+        await adminClient.ensureConnected()
+        console.log('deploying new community...')
+        community = await adminClient.deployCommunity({
+            provider: testProvider,
+        })
+        console.log(`Going to deploy to ${community.address}`)
+        await community.deployed()
+        console.log(`Deployment done for ${community.address}`)
+        await community.isReady(2000, 200000)
+        console.log(`Community ${community.address} is ready to roll`)
+        await adminClient.createSecret(community.address, 'secret', 'CommunityEndpoints test secret')
+    }, 300000)
+
+    afterAll(async () => adminClient.ensureDisconnected())
+    afterAll(() => testProvider.removeAllListeners())
+
+    describe('Admin', () => {
+        const memberAddressList = [
+            '0x0000000000000000000000000000000000000001',
+            '0x0000000000000000000000000000000000000002',
+            '0x000000000000000000000000000000000000bEEF',
+        ]
+
+        it('can add and remove members', async () => {
+            console.log('starting test')
+            await adminClient.communityIsReady(community.address, console.log)
+
+            await adminClient.addMembers(community.address, memberAddressList, testProvider)
+            await adminClient.hasJoined(community.address, memberAddressList[0])
+            const res = await adminClient.getCommunityStats(community.address)
+            assert.deepStrictEqual(res.memberCount, {
+                total: 3, active: 3, inactive: 0
+            })
```

Comment:
Going forward, where possible we should use [jest's `expect` API](https://jestjs.io/docs/en/expect) instead of `assert`.